lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/rest/TopologyRestService.java

```diff
@@ -64,14 +65,16 @@ public ResponseEntity getAllTopologyIdsOperational(Authentication authentication
             final Optional<NetworkTopology> readData =
                     tx.read(LogicalDatastoreType.OPERATIONAL, iid).get(TIMEOUT, TimeUnit.SECONDS);
 
-            if (readData.isPresent() && readData.get().getTopology() != null) {
-                final List<String> topology = readData.get().getTopology().values().stream()
-                        .map(topology1 -> topology1.getTopologyId().getValue())
-                        .collect(Collectors.toList());
-                return ResponseEntity.ok(topology);
-            } else {
-                return ResponseEntity.ok(Collections.emptyList());
```

Comment:
What is the problem here? I do not see any

Reply:
done, marked as false-positive